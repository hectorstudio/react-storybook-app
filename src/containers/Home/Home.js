import React, { Component } from 'react';

class Home extends Component {
  render() {
    return (
      <div>
        <div>Home Page</div>
      </div>
    );
  }
}

export default Home;
