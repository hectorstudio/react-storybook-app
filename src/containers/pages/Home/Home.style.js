import styled from 'styled-components';

export const HomeWrapper = styled.div`
  display: block;
`;
