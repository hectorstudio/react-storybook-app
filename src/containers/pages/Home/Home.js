import React, { Component } from 'react';
import { Row, Col } from 'antd';

import { HomeWrapper } from './Home.style';
import WalletView from '../../WalletView';
import ActionView from '../../ActionView';

class Home extends Component {
  render() {
    return (
      <HomeWrapper>
        <Row gutter={32}>
          <Col span={6}>
            <WalletView />
          </Col>
          <Col span={18}>
            <ActionView />
          </Col>
        </Row>
      </HomeWrapper>
    );
  }
}

export default Home;
