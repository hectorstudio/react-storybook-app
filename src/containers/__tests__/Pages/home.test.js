import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Home from '../../Home';

describe('Home Page', () => {
  it('renders with default props', () => {
    const wrapper = shallow(<Home />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
