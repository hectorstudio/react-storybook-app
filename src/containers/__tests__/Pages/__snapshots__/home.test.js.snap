// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders with default props 1`] = `
<styled.div>
  <Row
    gutter={32}
  >
    <Col
      span={6}
    >
      <WalletView />
    </Col>
    <Col
      span={18}
    >
      <ActionView />
    </Col>
  </Row>
</styled.div>
`;
